### PR TITLE
Update setup.rst

### PR DESCRIPTION
**Pull request by Goodiec**
This PR improves the setup page and is part of [issue #41](https://github.com/jupyterhub/outreachy/issues/41)

### CODE REVIEW

File: docs/source/contributing/setup.rst [hunk 1]
```diff
@@ -137,7 +135,7 @@ To simplify testing of JupyterHub, it’s helpful to use
 authenticator and SimpleLocalProcessSpawner instead of the default spawner.
 
 There is a sample configuration file that does this in
-``testing/jupyterhub_config.py``. To launch jupyterhub with this
+``testing/jupyterhub_config.py``. To launch Jupyterhub with this
```
Comment (minrk) [inline docs/source/contributing/setup.rst:138]:
```suggestion
``testing/jupyterhub_config.py``. To launch JupyterHub with this
```

**Review by minrk — COMMENTED** [review]
Thank you for the suggestions! I made one inline suggestion, and then we can merge.

File: docs/source/contributing/setup.rst (hunk 1 above)
Comment (GeorgianaElena) [reply docs/source/contributing/setup.rst:138]: @Goodiec, no worry, it's hard to spot a small difference of only one letter in these diffs.

`JupyterHub` has the upper letter `H` in the middle of the name, from `Hub`. 
Min's suggestion is to change  the name to `JupyterHub` (notice the upper letter `H`). In your commit it's currently `Jupyterhub` with the small letter `h`.

Comment (Goodiec) [reply docs/source/contributing/setup.rst]: Thank you @GeorgianaElena, I've made the correction.

Comment (GeorgianaElena) [reply docs/source/contributing/setup.rst]: > Also, is it Ok if I help review the PRs of others?

Of course, that would be great! Thanks @Goodiec

**Pull request merged**